### PR TITLE
Added ability to change color of accent, fixed installer service dialog

### PR DESCRIPTION
**Pull request by J0nathan550**
Fixes issue: #65 
Allows to change the accent of selected elements in FluentAvalonia.
The item is translated on all languages.

accent

### CODE REVIEW

File: SIT.Manager.Avalonia/ViewModels/SettingsPageViewModel.cs [hunk 1]
```diff
@@ -133,4 +141,14 @@ public partial class SettingsPageViewModel : ViewModelBase
         _localizationService.Translate(value);
         Config.CurrentLanguageSelected = value.Name;
     }
+
+    partial void OnCurrentAccentColorChanged(Color? value)
+    {
+        if (faTheme == null) faTheme = Application.Current?.Styles.OfType<FluentAvaloniaTheme>().FirstOrDefault();
```
Comment (Plootie) [inline SIT.Manager.Avalonia/ViewModels/SettingsPageViewModel.cs:147]: I've never observed this trigger, Is this needed?

**Review by Plootie — CHANGES_REQUESTED** [review]
Only request is that you change the binding to `Config.CurrentAccentColor` and apply the accent inside the config changed event similar to how the console font color is handled

File: SIT.Manager.Avalonia/ViewModels/SettingsPageViewModel.cs (hunk 1 above)
Comment (J0nathan550) [reply SIT.Manager.Avalonia/ViewModels/SettingsPageViewModel.cs:147]: > I've never observed this trigger, Is this needed?

For the theme specificly yes, I dig for one hour trying to get the FluentAvaloniaTheme. Lucky me, I found it.

**Comment by J0nathan550** [comment]
Refactored. Also fixed one dialog in Installer Service, I was trying to play game with my brother, but noticed that when he had wrong version of Tarkov it pop up him a dialog that would not give him brief dialog that the current version of tarkov is downgraded, plus there was no button at all to close the dialog which locked application forever. Translated it, and fix it.

**Review by Plootie — APPROVED** [review]

**Pull request merged**